### PR TITLE
Implemented default projector selection

### LINKED ISSUE #1062: Not possible to set projection defaults in a projector's edit view
…/projectors/detail/…

**Describe the bug**
It is currently not possible to set projection defaults in a projector's edit view. The drop-down menu does not appear or is not available.

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #1062

### CODE REVIEW

File: client/src/app/site/pages/meetings/pages/projectors/components/projector-edit-dialog/components/projector-edit-dialog/projector-edit-dialog.component.html [hunk 1]
```diff
@@ -93,20 +93,20 @@ <h3>{{ 'Resolution and size' | translate }}</h3>
...
+            <h3>{{ 'Projection defaults' | translate }}</h3>
+            <mat-form-field>
+                <mat-select
+                    formControlName="projectiondefault_ids"
+                    placeholder="{{ 'Projection defaults' | translate }}"
+                    [multiple]="true"
+                >
+                    <mat-option *ngFor="let pd of projectionDefaultOptions" [value]="pd.key">
+                        {{ pd.text | translate }}
+                    </mat-option>
```
Comment (GabrielInTheWorld) [inline client/src/app/site/pages/meetings/pages/projectors/components/projector-edit-dialog/components/projector-edit-dialog/projector-edit-dialog.component.html:105]:
```suggestion
                    <mat-option *ngFor="let pd of projectionDefaultOptions | keyvalue" [value]="pd.key">
                        {{ pd.value | translate }}
                    </mat-option>
```

File: client/src/app/site/pages/meetings/pages/projectors/components/projector-edit-dialog/components/projector-edit-dialog/projector-edit-dialog.component.ts [hunk 2]
```diff
@@ -82,6 +85,14 @@ export class ProjectorEditDialogComponent extends BaseUiComponent implements OnI
         return this.data.projector || null;
     }
 
+    /**
+     * Options for the `Projection defaults`-selector
+     */
+    public readonly projectionDefaultOptions = Object.keys(ProjectiondefaultVerbose).map(key => ({
```
Comment (GabrielInTheWorld) [inline client/src/app/site/pages/meetings/pages/projectors/components/projector-edit-dialog/components/projector-edit-dialog/projector-edit-dialog.component.ts:91]: Remove

File: client/src/app/site/pages/meetings/pages/projectors/components/projector-edit-dialog/components/projector-edit-dialog/projector-edit-dialog.component.ts [hunk 3]
```diff
@@ -203,11 +222,86 @@ export class ProjectorEditDialogComponent extends BaseUiComponent implements OnI
         }
     }
 
+    /**
+     * Processes all relevant data in a meeting (such as the projection defaults)
+     * @param meeting the changed meeting
+     */
+    private handleMeeting(meeting: ViewMeeting): void {
```
Comment (GabrielInTheWorld) [inline client/src/app/site/pages/meetings/pages/projectors/components/projector-edit-dialog/components/projector-edit-dialog/projector-edit-dialog.component.ts:229]:
```suggestion
    private updateProjectorDefaults(meeting: ViewMeeting): void {
```

**Review by GabrielInTheWorld — CHANGES_REQUESTED** [review]

**Review by emanuelschuetze — APPROVED** [review]

**Comment by emanuelschuetze** [comment]
@GabrielInTheWorld Can we merge this?

**Review by GabrielInTheWorld — APPROVED** [review]

**Pull request merged**